packages/data/src/registry.js

```diff
@@ -198,14 +193,19 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 	// Deprecated
 	// TODO: Remove this after `use()` is removed.
 	function withPlugins( attributes ) {
-		return mapValues( attributes, ( attribute, key ) => {
-			if ( typeof attribute !== 'function' ) {
-				return attribute;
-			}
-			return function () {
-				return registry[ key ].apply( null, arguments );
-			};
-		} );
+		return Object.fromEntries(
+			Object.entries( attributes ).map( ( [ key, attribute ] ) => {
```

Comment:
Will always be a properly defined registry object:

https://github.com/WordPress/gutenberg/blob/3bc98f04633e09125cbd948f5c907c0e3ee5b7fe/packages/data/src/registry.js#L354

https://github.com/WordPress/gutenberg/blob/3bc98f04633e09125cbd948f5c907c0e3ee5b7fe/packages/data/src/registry.js#L312